parking-lot-help-request/README.md

```diff
@@ -0,0 +1,118 @@
+* [Set up a Twilio account and configure the data to route from Notehub to Twilio](#twilio-firmware).
+## Hardware
+
+The following hardware is required to run the Parking Lot Help Request application.
+
+* [Notecarrier A with pre-soldered headers](https://shop.blues.io/products/carr-al)
+* [Notecard LTE Cat-1](https://shop.blues.io/products/note-wbna-500) (for use in North America)
+* [Sparkfun RGB LED breakout - WS2812B](https://www.sparkfun.com/products/13282)
+* [Twidec 12mm momentary push button with pre-soldered wires](https://www.amazon.com/gp/product/B08JHW8BPV/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1)
+* [LiPo battery](https://www.adafruit.com/product/2011)
+* [Solar panel with JST connectors (any of these are compatible)](https://www.seeedstudio.com/catalogsearch/result/?q=solar%20panels)
+* [Breadboard](https://www.adafruit.com/product/64)
+* [Male to male jumper wires](https://www.adafruit.com/product/758)
+* [Break-away 0.1" pin strip male headers](https://www.adafruit.com/product/392)
+
+### Hardware Assembly 
+
+1. After purchasing your hardware, use the [Notecard and Notecarrier Quickstart documentation](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/) to assemble your Notecarrier A and Notecard - don't worry too much over connecting to Notehub: this document will cover it shortly.
+1. Break off 3 male pins in a strip and solder to each side of the Sparkfun LED breakout. (This [video from Adafruit](https://www.youtube.com/watch?v=Z0joOKaQ43A&ab_channel=AdafruitIndustries) provides a good example of how to solder male pins to a board like the one the LED comes in.)
+1. Cut a male jumper wire in half and use a wire stripper to strip at least 1/2" of coating off of both the ends of the jumper wire and each wire connected to the Twidec push button. Also snip off the soldered tip of the button's wires so the filaments are free floating.
+![Twidec button and jumper wire both stripped down to expose their wires for soldering](images/readme-wires-to-solder.jpg)
+_Notice that both the button and jumper wires are exposed for soldering, also there's a length of heat shrink on both button wires that will be used to cover the soldered portion of wire later._
+
+1. Use the "Western Union" X-style of wire wrapping to join the exposed jumper wires together with the Twidec wires and solder them together.
+![Freshly soldered jumper wire and button wire](images/readme-wires-soldered.jpg)
+_A helping hand tool made soldering the jumper wire and button wire together much easier._
+
+1. Optional: Add a small sleeve of heat shrink tubing to each wire before soldering together and after soldering's complete, slide the heat shrink over the exposed wires and use a heat gun to shrink the wrap and protect the wires from the elements. (A great video that shows how do all of this, [wires to heat shrink, is available here](https://www.youtube.com/watch?v=Zu3TYBs65FM&ab_channel=ChrisFix).)
```

Comment:
The first sentence in this needs more punctuation or a bit of a rewording.

Reply:
Good suggestion.